### PR TITLE
MVVM structure basics

### PR DESCRIPTION
**Pull request by mouseorcleg**
File + structures

### CODE REVIEW

**Comment by mouseorcleg** [comment]
Here is the end of tutorial & my solo journey begins

**Comment by mouseorcleg** [comment]
Probably it will be a journey to another tutorial, but anyway :)

File: Eye in the sky/ViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    @Published var temp: String = "Is it cold?"
+    @Published var description: String = "Or do I feel cold?"
+    @Published var timeZone: String = "What time is it?"
+    
+    init() {
+        fetchWeather()
+    }
+    
+    func fetchWeather() {
+        guard let url = URL(string: "https://api.openweathermap.org/data/2.5/weather?lat=51.51&lon=-0.13&appid=cc5f0b417551c77d68956a6336441d72&units=metric") else {
```
Comment (togramago) [inline Eye in the sky/ViewModel.swift:23]: 1. move all networking to a separate layer
2. move URL to a separate constant, and appid to separate constant
3. make API call to /weather with different param inputs. 
minor: API also supports looking by city name, param name is q

Comment (mouseorcleg) [reply Eye in the sky/ViewModel.swift:23]: Will do tomorrow (in the new pull request) 👍

**Pull request merged**